src/damage.cpp

```diff
@@ -305,6 +305,17 @@ damage_type dt_by_name( const std::string &name )
     return iter->second;
 }
 
+std::string name_by_dt( const damage_type &dt ) {
```

Comment:
I'm also not sure what you mean with creating a `static const std::string err_msg( "dt_not_found" )`.